generate.js

```diff
@@ -46,8 +48,10 @@ const template = `
   </head>
   <body>
     <h1>GCI Current Leaders</h1>
+    <small id="last-generated"></small>
+    <hr>
     <i>
-      The leading participants for each organization are listed alphabetically
+      The leading participants for each organization are listed alphabetically 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpsmvkva_5/generate.js
+++ b/tmp/tmpsmvkva_5/generate.js
@@ -51,7 +51,7 @@
     <small id="last-generated"></small>
     <hr>
     <i>
-      The leading participants for each organization are listed alphabetically 
+      The leading participants for each organization are listed alphabetically
       according to their "display name"
     </i>
     <div class="orgs">
```